utils/utils.py

```diff
@@ -17,3 +19,80 @@ def preprocess_data(X, y=None, remove_zerovar=True, standardize=True):
             X = StandardScaler().fit_transform(X)
 
     return X, y
+
+
+def prox_l1sorted(v, w):
```

Comment:
Consider using other names here. These are both proxes of the Sorted L1 Norm (which is the same as the OWL norm). How about `prox_isotonic()` and `prox_fast_stack()` (since it's called something similar to that in the SLOPE paper).